README.md

```diff
@@ -60,6 +60,8 @@ Good luck.
   - [What is PaaS?](https://azure.microsoft.com/en-us/overview/what-is-paas/)
   - [What is SaaS?](https://azure.microsoft.com/en-us/overview/what-is-saas/)
   - [Creating an Azure account](https://docs.microsoft.com/en-us/dotnet/azure/create-azure-account)
+  - [Types of Database on Azure](https://azure.microsoft.com/en-us/product-categories/databases/)
+    - [SQL vs NoSQL](https://www.xplenty.com/blog/the-sql-vs-nosql-difference/#:~:text=SQL%20databases%20are%20relational%2C%20NoSQL,dynamic%20schemas%20for%20unstructured%20data.&text=SQL%20databases%20are%20table%20based,graph%20or%20wide%2Dcolumn%20stores.)
```

Comment:
What is the extra part of the URL? 

is this the actual URL? https://www.xplenty.com/blog/the-sql-vs-nosql-difference/

Reply:
This is the actual URL that you have pointed out https://www.xplenty.com/blog/the-sql-vs-nosql-difference/.
will make the changes, thank you for the review